plenario/api/point.py

```diff
@@ -15,8 +15,8 @@
 from plenario.api.response import internal_error, bad_request, json_response_base, make_csv
 from plenario.api.response import geojson_response_base, form_csv_detail_response, form_json_detail_response
 from plenario.api.response import form_geojson_detail_response, add_geojson_feature
-from plenario.api.validator import Validator, DatasetRequiredValidator
-from plenario.api.validator import NoDefaultDatesValidator, validate
+from plenario.api.validator import Validator, DatasetRequiredValidator, NoGeoJSONDatasetRequiredValidator
+from plenario.api.validator import NoDefaultDatesValidator, validate, NoGeoJSONValidator
```

Comment:
You know you can split across lines so you can avoid having `from plenario.api.validator import` twice?